tracer/src/Datadog.Trace.Tools.Runner/RunHelper.cs

```diff
@@ -16,7 +16,8 @@ public static bool TryGetEnvironmentVariables(ApplicationContext applicationCont
                 invocationContext,
                 applicationContext.RunnerFolder,
                 applicationContext.Platform,
-                settings);
+                settings,
+                false);
```

Comment:
```suggestion
                reducePathLength: false);
```

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/4558/commits/eb51dd2e7046f8d90ae8e43a368ce9587b55f5d4